deploy/role.yaml

```diff
@@ -32,49 +34,45 @@ rules:
   - list
   - watch
   - patch
-  - update
 - apiGroups:
   - extensions
   - apps
   resources:
   - deployments
   - replicasets
   - daemonsets
-  - statefulsets
```

Comment:
this was added because older k8s had a quirk that required permission for daemonsets *and* statefulsets to work properly. Probably it's indeed time to remove it.